app/models/comment.rb

```diff
@@ -11,9 +11,11 @@ class Comment < ActiveRecord::Base
     'five stars'  => '5_stars'
   }
     def humanized_rating
-    RATINGS.invert[self.rating]
-  def send_comment_email
```

Comment:
This `def send_comment_email` was nested underneath `def humanized_rating`, when it shouldn't be.  It should be at the root level.